src/server/req_manager.c

```diff
@@ -1629,6 +1628,11 @@ mgr_sched_set(struct batch_request *preq)
 	CLEAR_HEAD(unsetlist);
 	plist = (svrattrl *)GET_NEXT(preq->rq_ind.rq_manager.rq_attr);
 	while (plist) {
+		if (strcmp(plist->al_atopl.name, ATTR_scheduling) == 0) {
+			only_scheduling++;
+		} else {
+			only_scheduling--;
+		}
```

Comment:
How about calling this not_scheduling, and check strcmp() != 0, and then set the variable?  You only really care you got something that wasn't scheduling.  It's probably a bit clearer.